### PR TITLE
added color to result text

### PR DESCRIPTION
**Pull request by ruspedpdx**
### Description of changes
 I updated the missing lines in the flexbox exercise for part one
 The html page was updated to help evaluate and style the input and output 
 I added a function to test a number to see if it is a palindrome

### Reason for the changes
 To complete the assignment according to the instructions

### Feedback requested
 Any feedback regarding style or functionality would be appreciated. I spent more time trying to figure out the .js file and didn't focus as much on the .html, so I'm sure there will be many things to add to a review.

### CODE REVIEW

**Review by colonelcandycorn — APPROVED** [review]
Everything works good. I know you mentioned that you didn't focus as much on the html so I noticed there is no difference between the section and the background. I set up eslint to run the airbnb linter and got these messages on your code: 

   4:32  error    'handleInput' was used before it was defined                                                                      no-use-before-define
   6:22  error    'e' is defined but never used                                                                                     no-unused-vars
   8:15  error    Use object destructuring                                                                                          prefer-destructuring
   9:14  error    Unexpected use of 'isNaN'. Use Number.isNaN instead https://github.com/airbnb/javascript#standard-library--isnan  no-restricted-globals
  22:9   warning  Unexpected console statement

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+    }
+
+    if (value.length > 0 && value === value.split("").reverse().join("")) {
+      log.textContent = "Yes. This is a palindrome!";
+      log.style.color = "#06402B";
+    } else {
+      log.textContent = "No. Try again.";
+      log.style.color = "#8B0000";
+    }
+  } catch (error) {
```
Comment (Ghaadah) [inline hw1/02-palindrome.js:20]: I like that you used the exception (throw and catch statements )to handle the error in the handleInput function. Great Code!

File: hw1/02-palindrome.html [hunk 2]
```diff
@@ -6,20 +6,23 @@
...
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
   <body>
     <main>
       <section>
```
Comment (Ghaadah) [inline hw1/02-palindrome.html:15]: The HTML file is missing the Bootstraps styles. I suggest adding class = "container" to the section to control its background color. [Bootstraps](https://getbootstrap.com/docs/5.0/utilities/background/)

**Review by Ghaadah — COMMENTED** [review]
Great Code! I added some comments and suggestions for your review.

**Pull request merged**